docs/config.md

```diff
@@ -0,0 +1,200 @@
+Once this is done the next step is to configure the service provider.
+
+#### Configuring the Service Provider for the Sample App
+
+1. [Start](https://docs.wso2.com/display/IS540/Running+the+Product) the Identity Server and log in. (If you have not already done so)
+2. In the `Identity` section under the `Main` tab, click `Add` under `Service Providers`.
+3. Enter `travelocity.com` in the `Service Provider Name` text box and click `Register`.
+   ![Adding Service Provider](img/add-service-provider.png "Adding Service Provider")
+4. In the `Inbound Authentication Configuration` section, click `Configure` under the `SAML2 Web SSO Configuration` section.
+5. Add the following line to the `/etc/hosts` file.
+   ```
+   127.0.0.1       wso2is.local
+   ```
+   > Some browsers do not allow you to create cookies for a naked hostname, such as localhost. Cookies are required when working with SSO. Therefore, to ensure that the SSO capabilities work as expected in this tutorial, you need to configure the etc/host file as explained in this step.
+5. Configure the sample application (travelocity) as the service provider.
+   * Issuer: `travelocity.com`
+   * Assertion Consumer URL: `http://wso2is.local:8080/travelocity.com/home.jsp`
+6. Enable the following options.
+   * Response Signing
+   * Single Logout
+   * Attribute Profile
+   * Include Attributes in the Response Always
+   
+   > The above options should be enabled or disabled according to your service provider. For travelocity, the relevant properties file (`travelocity.properties`) can be found inside the webapp `travelocy.com/WEB-INF/classes/`.
+7. Click `Register` to save the changes. Now you will be sent back to the Service Providers page.
+   ![Configuring SAML SSO](img/configure-saml-sso.png "Configuring SAML SSO")
+
+
+Follow the next few steps to add the password policy to the sample app
+
+1. Go to `Local and Outbound Authentication Configuration` section in the Service Providers page.
+2. Select the `Advanced Configuration` radio button option.
+3. Add the `basic` authentication as the first step and `Password Reset Enforcer` authentication as the second step.
+   * Select `User subject identifier from this step` under `basic` authentication.
+   * Select `Use attributes from this step` under `Password Reset Enforcer`.
+   ![Authentication Configuration](img/authentication-configuration.png "Authentication Configuration")
+4. Click `Update` to save the changes.
+
+#### Testing the Sample App
+
+Follow the steps given below to test the Password Reset Enforcer.
+
+> These steps should not be followed in a production environment. Therefore if you follow these steps make sure you change it back to the original state accordingly before moving to a production environment.
+
+1. In the `Identity` section under the `Main` tab, click `List` under `Claims`.
+2. Click `http://wso2.org/claims`.
+3. Click `Edit` next to `Last Password Update Time` claim.
```

Comment:
Cliam name should be "Last Password Update", not  "Last Password Update Time"

Reply:
Changed.